packages/ui-progress/src/ProgressBar/__examples__/ProgressBar.examples.tsx

```diff
@@ -31,28 +32,27 @@ export default {
     valueNow: [0, 40, 80, 100],
     renderValue: [
       null,
-      // @ts-expect-error ts-migrate(7031) FIXME: Binding element 'valueNow' implicitly has an 'any'... Remove this comment to see the full error message
-      // eslint-disable-next-line react/display-name
-      ({ valueNow, valueMax }) => (
-        <span>{Math.round((valueNow / valueMax) * 100)}</span>
-      )
+      ({
+        valueNow,
+        valueMax
+      }: {
+        valueNow: ProgressBarProps['valueNow']
+        valueMax: ProgressBarProps['valueMax']
+      }) => <span>{Math.round((valueNow! / valueMax!) * 100)}</span>
     ]
   },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  getComponentProps: (props) => {
+  getComponentProps: () => {
     return {
       screenReaderLabel: 'Passing students',
       valueMax
     }
   },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  getExampleProps: (props) => {
+  getExampleProps: (props: ProgressBarProps) => {
     return {
-      background: props.color.includes('inverse')
+      background: props.color!.includes('inverse')
         ? 'primary-inverse'
         : 'primary'
     }
   },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  filter: (props) => {}
+  filter: () => {}
```

Comment:
I think the filter can be deleted if not used.

Reply:
Honestly, I didn't know if the functions are needed or not, so thanks I know now